### DISCUSSION TITLE
PDK package for Debian Bullseye

### DISCUSSION THREAD

**Discussion by bjvrielink** [puppetlabs/pdk#1109] [Feature Request] [answered]
Where can I find a PDK package for Debian Bullseye (i.e. Debian 11)?

**Comment by sanfrancrisko** [comment]
Hi @bjvrielink , we currently do not support Debian 11. Whenever the Puppet Agent officially adds support we will then look to add this to the PDK too.

**Comment by sanfrancrisko** [comment, marked as answer]
Hi @bjvrielink - PDK 2.2.0 added support for Debian 11: https://puppet.com/try-puppet/puppet-development-kit/
